src/config.rs

```diff
@@ -135,12 +134,10 @@ impl SerialisableCertificate {
 
 impl Default for SerialisableCertificate {
     fn default() -> Self {
-        let cert = unwrap!(rcgen::generate_simple_self_signed(vec![
-            "MaidSAFE.net".to_string()
-        ]));
+        let cert = rcgen::generate_simple_self_signed(vec!["MaidSAFE.net".to_string()]).unwrap();
 
         Self {
-            cert_der: unwrap!(cert.serialize_der()).into(),
+            cert_der: cert.serialize_der().unwrap().into(),
```

Comment:
I guess we want to get rid of unwraps, not sure what's the best option since default() cannot return Result...

Reply:
Perhaps we cannot have default in this case? Should be possible to Engineer this out AFAIK.

Reply:
In this PR I only removed the `unwrap!` macro by replacing it with regular `unwrap`. I know we want to remove those as well, but maybe best to take that in a separate PR? I though @lionel1704 was on it?

Reply:
I have raised madadam/quic-p2p#1 to this branch already. I can update the PR if needed.